lib/sycamore/sycamore/reader.py

```diff
@@ -207,6 +208,7 @@ def pandas(self, dfs: Union[DataFrame, list[DataFrame]]) -> DocSet:
         scan = PandasScan(dfs)
         return DocSet(self._context, scan)
 
+    @requires_modules("opensearchpy", extra="opensearch")
```

Comment:
Do we need any of the requires other than here?  I think this is the entry point so it's sufficient to have it here.